Hestia/app/src/main/java/hestia/UI/dialogs/EnterRequiredInfoDialog.java

```diff
@@ -143,9 +143,9 @@ void pressCancel() {
 
     @Override
     void pressConfirm() {
-        new AsyncTask<Object, String, Integer>() {
+        new AsyncTask<Object, String, Boolean>() {
             @Override
-            protected Integer doInBackground(Object... params) {
+            protected Boolean doInBackground(Object... params) {
```

Comment:
We don't use the Boolean anywhere, might as well just return a Void object and return null;